stock_ownership_availability_rules/model/quant.py

```diff
@@ -35,47 +35,29 @@ def create(self, vals):
             vals['owner_id'] = (
                 location.partner_id.id or
                 location.company_id.partner_id.id or
-                Company.browse(
-                    Company._company_default_get('stock.quant')
-                ).partner_id.id
-            )
+                Company._company_default_get('stock.quant').partner_id.id)
 
         return super(Quant, self).create(vals)
 
     owner_id = fields.Many2one('res.partner', 'Owner',
```

Comment:
Maybe we could take advantage to put field declarations before CRUD methods:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models